### PR TITLE
Update static for applicant-frontend + emails templates

### PR DESCRIPTION
**Pull request by aggelinaml**
Hey Marc Antoine, I just added some things on the file. 
I had some comments/questions for some parts, but I didn't know to add them!
I think for now I am getting better and better in coding, haha! :)
The rest, we will talk on Friday!

### CODE REVIEW

File: applicant-frontend/src/static.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "ERROR_REQUIRED_QUESTIONS": "Please answer correctly to required questions",
+  "ERROR_REQUIRED_QUESTIONS": "Please answer correctly to required questions.",
   "ERROR_VALID_PASSWORD": "Please provide a valid password. (10 characters: at least one uppercase, lowercase and one number)",
-  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example +77123123123) no space",
-  "ERROR_VALID_EMAIL": "Please provide a valid email address",
-  "SUBMIT_FORM_VALIDATION": "Are you sure to submit this form? If you prefer to edit it later, save it.",
+  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example 6934303600) no space",
```
Comment (Marc-AntoineA) [inline applicant-frontend/src/static.json:4]:
```suggestion
  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example +3036934303600 for a Greek Number) no space",
```
We need the '+' in the answer

File: backend/emails/accepted/html.pug [hunk 2]
```diff
@@ -1,8 +1,9 @@
 // pug template for sending emails after application
 // test online with https://pughtml.com
 p Hi #{name},
-p We are glad to announce you that you will be a members of #{campaignName}!
+p We are glad to announce you that you will be a member of #{campaignName}!
+p Class starts on (insert date & time)! Be there, on time!
```
Comment (Marc-AntoineA) [inline backend/emails/accepted/html.pug:5]: For the first round, write directly the date & time. We could imagine add this date later on the database.

**Comment by Marc-AntoineA** [comment]
Thanks for the updates in all the templates!

For the `.pug` format, please have a look to the documentation (https://pugjs.org/api/getting-started.html) and use this website (https://pughtml.com/) to visualize the result (see at the end of the page, don't look at the html).

For example the template `application/html.pug` doesn't work now. (the `|` and `a` where here for a reason).
I'll update it myself before pushing it this week.

One last remark, when you want to do edits, do them on the branch 'develop', not on the branch 'master'.

**Comment by aggelinaml** [comment]
Hey Marc Antoine, I am really sorry about the a & I, I didn't know that. Zohir explained me. 
Apologies about it and sorry for the extra work!

Yes, Inigo (Erasmus Intern) showed me the develop brunch. I will use it in the future for other changes

File: applicant-frontend/src/static.json (hunk 1 above)
Comment (aggelinaml) [reply applicant-frontend/src/static.json:4]: I am not sure, they will get it the +30. They might not even knoe the country code. In the worst case scenarion, they will copy paste the suggestion as a number. But you mean that the platform cannot work without the + ???

Comment (aggelinaml) [reply applicant-frontend/src/static.json:4]: So, in the code you wrote, which version of phone number is acceptable?

File: backend/emails/accepted/html.pug (hunk 2 above)
Comment (aggelinaml) [reply backend/emails/accepted/html.pug:5]: So know I should edit the:
Campaign name (I guess it means the class name, e.g. 9th Coding Class o just Coding Class?)
Location (is Athens right?) 

These names, where should I change them? In these texts or in another database?

**Comment by Marc-AntoineA** [comment]
Not merged in the proper way but merged into develop branch

**Pull request closed without merging**